src/test/mochitest/browser_dbg-preview.js

```diff
@@ -65,4 +65,7 @@ add_task(async function() {
   await popupPreviewed;
   assertPopup(dbg, { field: "foo", value: "1", expression: "obj" });
   assertPopup(dbg, { field: "bar", value: "2", expression: "obj" });
+
+  await waitForLoadedObjects(dbg)
```

Comment:
is this part of the work for adding telemetry probes?

Reply:
I don't know either but it would be nice to clean it up before merging?